mlflow/store/file_store.py

```diff
@@ -190,10 +190,11 @@ def list_experiments(self, view_type=ViewType.ACTIVE_ONLY):
         return experiments
 
     def _create_experiment_with_id(self, name, experiment_id, artifact_uri):
+        if not artifact_uri:
+            raise MlflowException(
+                "Experiment artifact URI must be a non-empty string, found: %s" % artifact_uri)
```

Comment:
`artifact_uri` is an empty-string or `None`

Reply:
Can you clarify?